api/src/main/java/org/pmiops/workbench/utils/CohortMapper.java

```diff
@@ -0,0 +1,27 @@
+package org.pmiops.workbench.utils;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.ReportingPolicy;
+import org.pmiops.workbench.db.model.DbCohort;
+import org.pmiops.workbench.db.model.DbUser;
+import org.pmiops.workbench.model.Cohort;
+
+@Mapper(
+    componentModel = "spring",
+    uses = {CommonMappers.class},
+    unmappedTargetPolicy = ReportingPolicy.IGNORE)
+public interface CohortMapper {
+  DbCohort clientToDbModel(Cohort source);
```

Comment:
we've been using `api` to refer to the client/api models.

Also, I kind of like just naming it as `toDb` or `toApi`. It's short and captures most of the context I care about and makes sense even when overloaded.

Reply:
Ehh, there are APIs all over the place. Whose api are we talking about? It's like saying "service" or "computer"; it's kind of useless. (BTW, I have a plan to rename all those classes with a prefix, so we'd have `ApiCohort` or `ClientApiCohort`, which may help.

Also, maybe propose a convention and do it globally? Most of CH's methods are using client. I'd prefer to make sure the whole team is behind a convention before switching, and I don't mind helping with big PRs that align conventions.